<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Raw sprite viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>531</height>
     </rect>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="DiagCrossPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxWidth">
    <property name="geometry">
     <rect>
      <x>562</x>
      <y>10</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>16384</number>
    </property>
    <property name="value">
     <number>16</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelAnchoSpr">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ancho sprite</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOffset">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Offset</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOffset">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>0x0000</string>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelFileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>631</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;No file&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFormato">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Formato</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxFmt">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelEscala">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Escala</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxEscala">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>43</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditHexDmp">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>351</width>
      <height>451</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditOffset</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>onOffsetChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxWidth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onWidthChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxFmt</sender>
   <signal>activated(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onImgFormatChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>752</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxEscala</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onScaleChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOffsetChange()</slot>
  <slot>onWidthChange(int)</slot>
  <slot>onImgFormatChange(int)</slot>
  <slot>onScaleChange(int)</slot>
 </slots>
</ui>
